.babelrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I don't particularly like supporting old versions of anything, but in this case it's not a big enough difference to matter. Let's keep it as its own file for now. When babel 7 is out, that'll be the time to go "yeah no seriously stop using babel 5".